### DISCUSSION TITLE
Accepting PRs for other modules?

### DISCUSSION THREAD

**Discussion by nad-au** [testcontainers/testcontainers-node#533] [Q&A] [answered]
Module support in node is bit behind other languages. Are you accepting contributions for other custom modules? For example, we're using https://memgraph.com/ which is very similar to Neo4j.

**Comment by cristianrgreco** [comment, marked as answer]
Hi @neildobson-au, yes, contributions of other modules are very welcome! 

You can find some docs about [creating custom containers](https://node.testcontainers.org/features/containers/#creating-a-custom-container) and can also reference the [existing Neo4j module](https://github.com/testcontainers/testcontainers-node/blob/main/src/modules/neo4j/neo4j-container.ts). If you do create a module, please also [add some docs](https://github.com/testcontainers/testcontainers-node/tree/main/docs/modules).

If I can help out just let me know :+1:
